src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+sidebarDepth: 2
+summaryPoint1: Centralized identity systems are fragmented, poorly secured, and offer minimal privacy protection
+summaryPoint2: Decentralized identity increases personal control of one's identity information.
+summaryPoint3: Decentralized ID projects built on Ethereum can revolutionize identity management. 
+---
+
+# What are identity, identifiers and attestations? {#what-are-identity-identifiers-attestations}
+
+## What is identity {#what-is-identity}
+
+Identity means an individual's sense of self, defined by a set of unique characteristics. Identity refers to the fact of being an _individual_, i.e., a distinct human entity. Identity could also refer to other non-human entities, such as an organization or authority.
+
+## What are identifiers? {#what-are-identifiers}
+
+An identifier is a piece of information that can be used to distinguish a person's identity. Common identifiers include the following: 
+- Name
+- Social security number/tax ID number 
+- Mobile number 
+- Date and place of birth
+- Digital identification credentials, e.g., email addresses, usernames, avatars
+
+It is important to understand that in these traditional examples of identifiers that _issuance_ and _control_ over these identifiers are maintained by a central agency. For example, legal names require permission from a government for issuance or before being changed, and changes to your social media handle require permission from the platform. 
+
+## What are attestations? {#what-are-attestations}
+ 
+An attestation refers to a claim made by an entity about another entity. An example attestation is a driver's license issued by the Department of Motor Vehicles (DMV). A driver's license, which provides a specific claim (that the holder has legal permission and is competent enough to drive a car) is made by one entity (the DMV) about another entity (the driver). 
+
+Attestations are different from identifiers. An attestation _contains_ identifiers to reference a particular identity, and makes a claim about an attribute related to this identity. The driver's license mentioned earlier, for example, has a mix of both identifiers (such as name, date of birth, and address), as well as an attestation about this individuals legal right to operate a vehicle.
+
+## What are decentralized identifiers? {#what-are-decentralized-identifiers}
+
+As mentioned, traditional identifiers like legal name, email address or Twitter handle are all reliant on intermediaries such as a governments, email providers or social media platforms. Decentralized identifiers (DIDs) are identifiers that are not issued, managed, or controlled by any central authority.
+
+Decentralized identifiers are issued, held, and controlled by individuals. An [Ethereum account](/developers/docs/accounts/) is an example of a decentralized identifier. Anyone is free to create as many accounts as they wish, without permission from anyone, and without need to store this identifier in a central registry.
```

Comment:
Yeah, that's kind of the idea of decentralized/self-sovereign identity (i.e., you can issue yourself an identifier without relying on any entity). I think creating an Ethereum account would qualify as self-issuing digital identifiers since you can do it without inputs from others. 

Here's a definition of decentralized identity for context: 

*"What is a decentralized identity? Decentralized identity is based on a trust framework for identity management. It allows users to generate and control their own digital identity without depending on a specific service provider."*